CONTRIBUTING.rst

```diff
@@ -22,13 +22,10 @@ Anaconda Installer Branching Policy (the long version)
 The basic premise is that there are the following branches:
 
 - master
-- unstable
 - <next fedora number>-release
 - <next fedora number>-devel
 
-``Master`` branch never waits for any release-related processes to take place. The spec file will remain there to track dependencies. ``Master`` branch is *not* associated with Fedora Rawhide builds anymore. Its purpose is to function purely as an upstream branch.
-
-The ``unstable`` branch is used for making periodic Anaconda releases for Rawhide (or possibly anyone else wanting to taste the cutting edge).
+``Master`` branch never waits for any release-related processes to take place. The spec file will remain there to track dependencies. ``Master`` branch is associated with Fedora Rawhide builds.
```

Comment:
Suggestion:

``Master`` branch never waits for any release-related processes to take place and is used for Fedora Rawhide Anaconda builds.

Reply:
Fixed.